org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/contentassist/AbstractContentProposalProvider.java

```diff
@@ -130,7 +130,13 @@ protected StyledString getStyledDisplayString(IEObjectDescription description) {
 	}
 	
 	protected StyledString getStyledDisplayString(EObject element, String qualifiedName, String shortName) {
-		return new StyledString(getDisplayString(element, qualifiedName, shortName));
+        StyledString styledString = new StyledString(getDisplayString(element, qualifiedName, shortName));
```

Comment:
wont be called on xbase langs. is there any lang to test this?

Reply:
do we need to change org.eclipse.xtext.xtext.ui.contentassist.XtextProposalProvider.getStyledDisplayString(EObject, String, String) too?

Reply:
![Bildschirmfoto 2021-05-21 um 06 44 28](https://user-images.githubusercontent.com/238736/119083735-d1597800-ba00-11eb-93c6-bcbd5810f994.png)

sure DECORATOR is right and not QUALIFIER?

Reply:
You could be right. Maybe the QUALIFIER is the better one.

Reply:
I updated the PR to use the `StyledString.QUALIFIER_STYLER`